services/client/storage.go

```diff
@@ -131,7 +131,7 @@ func (s *client) Placement(sid services.ServiceID) (services.Placement, int, err
 
 	p, err := placement.NewPlacementFromProto(&placementProto)
 	if p != nil {
-		p.SetVersion(v.Version())
+		p = p.SetVersion(v.Version())
```

Comment:
This is not a bug, I changed the semantic for the setters on placement, now every setter will create a new placement, will gradually change all the setters in m3cluster to that semantic.